recipes-debian/edac-utils/edac-utils_debian.bb

```diff
@@ -0,0 +1,56 @@
+# base recipe: meta-openembedded-warrior/meta-oe/recipes-bsp/edac-utils/edac-utils_git.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+SUMMARY = "Userspace helper for Linux kernel EDAC drivers"
+HOMEPAGE = "https://github.com/grondo/edac-utils"
+SECTION = "Applications/System"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=94d55d512a9ba36caa9b7df079bae19f"
+
+DEPENDS = " sysfsutils"
+
+SRCREV = "f9aa96205f610de39a79ff43c7478b7ef02e3138"
+PV = "0.18+git${SRCPV}"
+
+inherit debian-package
+require recipes-debian/sources/edac-utils.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/grondo-edac-utils-ea4d2e7"
+
+FILES_${PN} += "\
+    ${systemd_system_unitdir} \
+"
+
+SRC_URI += " \
+    file://edac.service \
+"
+
+inherit autotools-brokensep
+
+do_configure_prepend () {
+    touch ${S}/ChangeLog
+    ${S}/bootstrap
+}
+
+RDEPENDS_${PN}_x86 = "dmidecode"
```

Comment:
aarch64 and other architectures depend on dmidecode package, do we need dmidecode package too?

Reply:
We will use dmidecoded in poky.
This time, the decision is that it will not be used at runtime.

Reply:
Okay. I got it.